x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -508,7 +508,7 @@ from employees | stats c = count(*) by gender | sort gender;
 c:l | gender:s
 33  | F 
 57  | M
-10  | null
+#10  | null
```

Comment:
@nik9000 There's a discrepancy between EsqlActionIT (running in Lucene) and CsvTests - the former don't have a null group for gender while the latter do hence why the same csv-spec fails for one but passes for the other.
I've commented this line to make EsqlActionIT pass but it will make CsvTests fail.
Since the count works inside the group, I wonder if null group is either discarded when querying against Lucene or if the underlying filter doesn't work properly (ignores missing values somehow).
Any ideas?

Reply:
FTR, this was caused by https://github.com/elastic/elasticsearch/issues/100109